doc/reference/lora_lorawan/index.rst

```diff
@@ -1,26 +1,53 @@
-.. _lorawan_api:
+.. _lora_lorawan_api:
 
-LoRaWAN
-#######
+LoRa and LoRaWAN
+################
 
 Overview
 ********
 
-Long Range Wide Area Network (LoRaWAN) is a network layer protocol based on
-the LoRa PHY.
+LoRa (abbrev. for Long Range) is a proprietary low-power wireless
+communication protocol developed by the `Semtech Corporation`_.
+
+LoRa defines the physical layer of the protocol stack, which uses
```

Comment:
LoRa acts as the physical layer (PHY) based on the Chirp Spread Spectrum (CSS) modulation technique.

Reply:
Done.